README.md

```diff
@@ -32,11 +32,9 @@
 | cosmos           | [azure-spring-boot-starter-cosmos:3.9.0]                                                       | [cosmos-multi-database-single-account](cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-single-account)              |
 | cosmos           | [azure-spring-boot-starter-cosmos:3.9.0]                                                       | [cosmos](cosmos/azure-spring-boot-starter-cosmos/cosmos)                                                                          |
 | eventhubs        | [azure-spring-cloud-starter-eventhubs-kafka:2.9.0]                                             | [eventhubs-kafka](eventhubs/azure-spring-cloud-starter-eventhubs-kafka/eventhubs-kafka)                                           |
-| eventhubs        | [azure-spring-cloud-starter-eventhubs:2.9.0]                                                   | [eventhubs-integration](eventhubs/azure-spring-cloud-starter-eventhubs/eventhubs-integration)                                     |
-| eventhubs        | [azure-spring-cloud-starter-eventhubs:2.9.0]                                                   | [eventhubs-messaging](eventhubs/azure-spring-cloud-starter-eventhubs/eventhubs-messaging)                                         |
-| eventhubs        | [azure-spring-cloud-starter-eventhubs:2.9.0]                                                   | [eventhubs-operation](eventhubs/azure-spring-cloud-starter-eventhubs/eventhubs-operation)                                         |
-| eventhubs        | [azure-spring-cloud-stream-binder-eventhubs:2.9.0]                                             | [eventhubs-binder](eventhubs/azure-spring-cloud-stream-binder-eventhubs/eventhubs-binder)                                         |
-| eventhubs        | [azure-spring-cloud-stream-binder-eventhubs:2.9.0]                                             | [eventhubs-multibinders](eventhubs/azure-spring-cloud-stream-binder-eventhubs/eventhubs-multibinders)                             |
+| eventhubs        | [spring-cloud-azure-starter-integration-eventhubs:4.0.0-beta.1]                                                   | [eventhubs-integration](eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration)                                     |
```

Comment:
please update the link tags in the bottom of this file as well, and the link value should be udpated after release.

Reply:
updated